dash/dash.py

```diff
@@ -217,8 +262,12 @@ def _relative_url_path(relative_package_path='', namespace=''):
                         srcs.append(url)
             elif 'absolute_path' in resource:
                 raise Exception(
-                    'Serving files form absolute_path isn\'t supported yet'
+                    'Serving files from absolute_path isn\'t supported yet'
                 )
+            elif 'asset_path' in resource:
+                static_url = flask.url_for('assets.static',
+                                           filename=resource['asset_path'])
+                srcs.append(static_url)
```

Comment:
I did cache busting before with webpack and an extension. Was basically a template render that replaced the hash part of a filename with a new one. 

In dash, I think it would be kinda hard to do that, but we could have a watcher on the asset folder, copy those assets with a filename including the hash to a temp static folder, keep the hash in a dict with the path as key and serve the file with the hash formatted in. When a file change, copy it to the temp folder with a new hash and put the new hash as the value for the path. Could also tell the browser to reload while we're at it.

Reply:
Yeah that's a good idea. I'm a little worried about introducing a temp folder, I feel like users won't know why it's there or if they should commit it, etc.

Instead of having a watcher, could we just call this function on every page load (while in `dev` mode) and get the latest timestamp of the file? And then if we're not in `dev` mode, we could store the timestamps on the first page load?

Reply:
Cool, sounds like there are a couple of options. So, let's create a new GitHub issue about this and tackle it in a subsequent PR